src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,40 @@
+(ns status-im2.contexts.wallet.events
+  (:require [utils.re-frame :as rf]
+            [taoensso.timbre :as log]
+            [utils.security.core :as security]
+            [native-module.core :as native-module]))
+
```

Comment:
@ilmotta would appreciate a review on these events 👍

Reply:
also do we have a way to test these rpc requests? 🤔 -  I can add them here if there's a good example?
cc @cammellos

Reply:
> @ilmotta would appreciate a review on these events 

Will do :)

> also do we have a way to test these rpc requests?

The integration tests can make actual JSON RPC calls. If you would like to be sure more complex flows are correct I believe that's the place. If you want to venture in those tests, it would be good to write them in a separate file than `status-im.integration-test`, that one is crowded.